internal/service/infra_mgmt/model_infra_service.go

```diff
@@ -0,0 +1,181 @@
+	Name            types.String      `tfsdk:"name"`
+	PoolId          types.String      `tfsdk:"pool_id"`
+	ServiceType     types.String      `tfsdk:"service_type"`
+	Tags            types.Map         `tfsdk:"tags"`
+	UpdatedAt       timetypes.RFC3339 `tfsdk:"updated_at"`
+}
+
+var InfraServiceAttrTypes = map[string]attr.Type{
+	"configs":          types.ListType{ElemType: types.ObjectType{AttrTypes: InfraServiceHostConfigAttrTypes}},
+	"created_at":       timetypes.RFC3339Type{},
+	"description":      types.StringType,
+	"desired_state":    types.StringType,
+	"desired_version":  types.StringType,
+	"id":               types.StringType,
+	"interface_labels": types.ListType{ElemType: types.StringType},
+	"name":             types.StringType,
+	"pool_id":          types.StringType,
+	"service_type":     types.StringType,
+	"tags":             types.MapType{ElemType: types.StringType},
+	"updated_at":       timetypes.RFC3339Type{},
+}
+
+var InfraServiceResourceSchemaAttributes = map[string]schema.Attribute{
+	"configs": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: InfraServiceHostConfigResourceSchemaAttributes,
+		},
+		Computed:            true,
+		MarkdownDescription: "List of Host-specific configurations of this Service.",
+	},
+	"created_at": schema.StringAttribute{
+		CustomType:          timetypes.RFC3339Type{},
+		Computed:            true,
+		MarkdownDescription: "Timestamp of creation of Service.",
+	},
+	"description": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "The description of the Service (optional).",
+	},
+	"desired_state": schema.StringAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             stringdefault.StaticString("stop"),
+		MarkdownDescription: "The desired state of the Service. Should either be `\"start\"` or `\"stop\"`.",
+		Validators: []validator.String{
+			stringvalidator.OneOf("start", "stop"),
+		},
+	},
+	"desired_version": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "The desired version of the Service.",
+	},
+	"id": schema.StringAttribute{
+		Computed:            true,
+		MarkdownDescription: "The resource identifier.",
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.UseStateForUnknown(),
+		},
+	},
+	"interface_labels": schema.ListAttribute{
+		ElementType:         types.StringType,
+		Optional:            true,
+		MarkdownDescription: "List of interfaces on which this Service can operate. Note: The list can contain custom interface labels (Example: `[\"WAN\",\"LAN\",\"label1\",\"label2\"]`)",
+	},
+	"name": schema.StringAttribute{
+		Required:            true,
+		MarkdownDescription: "The name of the Service (unique).",
+	},
+	"pool_id": schema.StringAttribute{
+		Required:            true,
+		MarkdownDescription: "The resource identifier.",
+	},
+	"service_type": schema.StringAttribute{
```

Comment:
The values that this field support, can we add these to the docs?

Reply:
added types in the description

Reply:
Thank you